src/OnchainKitConfig.ts

```diff
@@ -5,7 +5,9 @@ import { OnchainKitConfig, SetOnchainKitConfig } from './types';
 // but only acccessed through the get and set functions.
 export const ONCHAIN_KIT_CONFIG: OnchainKitConfig = {
   address: null,
+  apiKey: '',
   chain: baseSepolia,
+  rpcUrl: '',
```

Comment:
I think is best we have `null` as negative. So the type here is either `string | null`.